backend/de.metas.fresh/de.metas.fresh.base/src/test/java/de/metas/util/lang/All_RepoIdAware_Classes_Test.java

```diff
@@ -68,9 +68,12 @@ public class All_RepoIdAware_Classes_Test
 			.skip(de.metas.bpartner.BPartnerBankAccountId.class)
 			.skip(de.metas.bpartner.user.role.UserAssignedRoleId.class)
 			//
+			.skip(de.metas.calendar.standard.YearId.class)
```

Comment:
that's kind of legacy. instead of adding it here, pls annotate ModuleConfigId with de.metas.util.lang.RepoIdAwares.SkipTest annotation

Reply:
done.